src/util-exception-policy.c

```diff
@@ -111,12 +134,21 @@ enum ExceptionPolicy ExceptionPolicyParse(const char *option, const bool support
                     policy == EXCEPTION_POLICY_BYPASS_FLOW) {
                 SCLogWarning(SC_WARN_COMPATIBILITY,
                         "flow actions not supported for %s, defaulting to \"ignore\"", option);
-                policy = EXCEPTION_POLICY_IGNORE;
+                policy = EXCEPTION_POLICY_NOT_SET;
             }
         }
 
     } else {
-        SCLogConfig("%s: ignore", option);
+        /* Exception Policy was not defined individually */
+        enum ExceptionPolicy master_policy = GetMasterExceptionPolicy(option);
+        if (master_policy == EXCEPTION_POLICY_NOT_SET) {
+            SCLogConfig("%s: ignore", option);
+        } else {
+            /* If the master switch was set and the Exception Policy option was not
+            individually set, use the defined master Exception Policy */
+            SCLogConfig("%s: defined via Exception Policy master switch", option);
```

Comment:
this should tell us more about the actual setting. What is the policy here?
```
[382533] 20/12/2022 -- 17:57:46 - (util-exception-policy.c:124) <Config> (ExceptionPolicyParse) -- exception-policy: auto
[382533] 20/12/2022 -- 17:57:46 - (util-exception-policy.c:149) <Config> (ExceptionPolicyParse) -- app-layer.error-policy: defined via Exception Policy master switch
[382533] 20/12/2022 -- 17:57:46 - (util-exception-policy.c:149) <Config> (ExceptionPolicyParse) -- defrag.memcap-policy: defined via Exception Policy master switch
[382533] 20/12/2022 -- 17:57:46 - (util-exception-policy.c:149) <Config> (ExceptionPolicyParse) -- flow.memcap-policy: defined via Exception Policy master switch
[382533] 20/12/2022 -- 17:57:46 - (util-exception-policy.c:149) <Config> (ExceptionPolicyParse) -- stream.memcap-policy: defined via Exception Policy master switch
[382533] 20/12/2022 -- 17:57:46 - (util-exception-policy.c:149) <Config> (ExceptionPolicyParse) -- stream.reassembly.memcap-policy: defined via Exception Policy master switch
[382533] 20/12/2022 -- 17:57:46 - (util-exception-policy.c:149) <Config> (ExceptionPolicyParse) -- stream.midstream-policy: defined via Exception Policy master switch
```

How about:
```
stream.midstream-policy: drop-flow (defined via 'exception-policy' master switch)
```